src/codegen/ast_interpreter.cpp

```diff
@@ -69,7 +69,8 @@ union Value {
 
 class ASTInterpreter {
 public:
-    typedef llvm::DenseMap<InternedString, Box*> SymMap;
+    typedef llvm::DenseMap<InternedString, int> SymMap;
+    typedef std::vector<Box*> SymVector;
```

Comment:
Maybe we should factor this out a separate datastructure?  ex a "ContiguousMap" (or whatever) that abstracts these operations.